Source/Core/UICommon/UICommon.cpp

```diff
@@ -366,63 +366,6 @@ bool TriggerSTMPowerEvent()
   return true;
 }
 
-#if defined(HAVE_XRANDR) && HAVE_X11
-void EnableScreenSaver(Window win, bool enable)
-#else
-void EnableScreenSaver(bool enable)
-#endif
-{
-// Inhibit the screensaver. Depending on the operating system this may also
-// disable low-power states and/or screen dimming.
-
-#if defined(HAVE_X11) && HAVE_X11
-  if (SConfig::GetInstance().bDisableScreenSaver)
-  {
-    X11Utils::InhibitScreensaver(win, !enable);
```

Comment:
If X11Utils::Inhibit isn't being called, then it should be deleted. (It's inlined in the fallback path, correct?)